docs/requirements.txt

```diff
@@ -0,0 +1,2 @@
+sphinx-rtd-theme
```

Comment:
Since the docs are installed on rtf, do you really need to include the theme for the docs?